src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import core.basesyntax.controller.Controller;
+import core.basesyntax.lib.Injector;
+
+public class Main {
+    public static void main(String[] args) {
+        try {
+            Injector.injectDaoDependency();
+            Controller.controlProcess();
+        } catch (IllegalArgumentException e) {
```

Comment:
do we need two the same catch block here?

Reply:
ok, thought if we have 2 different exeptions - we must hande they in 2 different ways?

Reply:
one exeption from method set(), another (aka our custom exeption) when dao not found